helix-core/src/main/java/org/apache/helix/manager/zk/CallbackHandler.java

```diff
@@ -245,6 +294,17 @@ public CallbackHandler(HelixManager manager, RealmAwareZkClient client, Property
     parseListenerProperties();
 
     init();
+
+    /**
+     * Async batch read was designed that if an exception is thrown, the operation fails and returns a partial result.
+     * The issue is separately addressed by checking read result in batch read calls [https://github.com/apache/helix/pull/974]
+     * and retry batch read for ZK connectivity issues [https://github.com/apache/helix/pull/970].
+     * We also add this one to periodical read stale messages from ZkServer in the case we don't see any event for a certain period.
+     */
+    _periodicTriggerInterval = periodicTriggerInterval;
+    if (_periodicTriggerInterval > 0) {
+      initRefreshTask();
```

Comment:
nit, pass the periodicTriggerInterval to initRefreshTask() and do the branch inside. That makes this code cleaner.